### PR TITLE
STORM-2432: Storm-Kafka-Client Trident Spout Seeks Incorrect Offset With UNCOMMITTED_LATEST Strategy

### PR DESCRIPTION
**Pull request by hmcl**
- Add check to execute first poll  offset strategy only for the first poll rather than always

### CODE REVIEW

**Comment by hmcl** [comment]
This patch must be ported onto master branch as well. Thanks.

**Comment by harshach** [comment]
+1

**Comment by satishd** [comment]
+1 LGTM

File: external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitter.java [hunk 1]
```diff
@@ -149,7 +150,7 @@ private void emitTuples(TridentCollector collector, ConsumerRecords<K, V> record
 
     /**
      * Determines the offset of the next fetch. For failed batches lastBatchMeta is not null and contains the fetch
-     * offset of the failed batch. In this scenario the next fetch will take place at the offset of the failed batch.
+     * offset of the failed batch. In this scenario the next fetch will take place at the offset of the failed batch + 1.
```
Comment (satishd) [inline external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitter.java]: minor nit: Just to make it clear.
>next fetch will take place at (offset of the failed batch + 1).

**Comment by satishd** [comment]
@hmcl I will take care of pushing these changes to master branch once this is merged into 1.x-branch.

**Comment by hmcl** [comment]
@satishd Done!

**Comment by hmcl** [comment]
This PR has already been merged by @satishd on commit 38f866c64c9bb1c55109a122c8ea3dc1a3b4b09b

**Pull request closed without merging**